README.md

```diff
@@ -933,6 +933,19 @@ this should be set to the authorization server's URL.
 The audience can also be specified using the `{optional-prefix}SF_AUDIENCE`
 environment variable.
 
+##### `expiration_offset`
+
+| Description | Valid Values | Required | Default |
+| --- | --- | --- | --- |
+| An offset duration (in _minutes_) to use when calculating the JWT `exp` claim | integer | N | 5 |
+
+This value of this parameter is added to the current time and the result is used
+as the value of the `exp` claim in the JSON Claims Set for the JWT. The value
+_must_ be a positive integer.
```

Comment:
Perhaps clearer?
```suggestion
An optional offset to increase the timeout of the `exp` claim in the JSON Claims Set for the JWT. Must be a positive integer.
- The `exp` will be calculated as `current_time + offset`
```

Reply:
Hey @jbeveland27 thanks for the comment.  While it is an optional config, I don't want it to sound like there is _no_ offset by default.  I think the original doc makes this slightly clearer even if it may not be entirely clear in general.  Going to go with that for now.